### PR TITLE
Change the preview banner message if master netlify preview

### PR DESCRIPTION
**Pull request by nickcolley**
Fixes https://github.com/alphagov/govuk-design-system/issues/193

Master

screen shot 2018-02-23 at 15 58 24

Pull request

screen shot 2018-02-23 at 15 58 48

### CODE REVIEW

File: views/partials/_banner.njk [hunk 1]
```diff
@@ -1,15 +1,24 @@
 {% from "phase-banner/macro.njk" import govukPhaseBanner %}
 
 {% if PULL_REQUEST %}
-  {% set pr_url = "https://github.com/alphagov/govuk-design-system/pull/" + REVIEW_ID %}
+  {% set prUrl = "https://github.com/alphagov/govuk-design-system/pull/" + REVIEW_ID %}
```
Comment (36degrees) [inline views/partials/_banner.njk]: Suggest this is either moved inside the `{% if REVIEW_ID %}` block or just omitted entirely and constructed within the `<a>` tag itself.

Comment (igloosi) [reply views/partials/_banner.njk]: would it be worth adding the branch name if no REVIEW_ID is present?

REPOSITORY_URL: URL to the Git repository the build pulls changes from.
BRANCH: Reference to check out after fetching changes from the Git repository.

Comment (nickcolley) [reply views/partials/_banner.njk]: Sounds interesting, I'll give it a go.

Comment (igloosi) [reply views/partials/_banner.njk]: this is your current branch url https://**fix-master-netlify**--govuk-design-system-preview.netlify.com/

**Pull request merged**